lib/ansible/modules/cloud/amazon/aws_ses_identity.py

```diff
@@ -409,6 +413,7 @@ def update_identity_notifications(connection, module):
     for notification_type in ('Bounce', 'Complaint', 'Delivery'):
         changed |= update_notification_topic(connection, module, identity, identity_notifications, notification_type)
         changed |= update_notification_topic_headers(connection, module, identity, identity_notifications, notification_type)
+        identity_notifications = get_identity_notifications(connection, module, identity)
```

Comment:
There should be no need to do this in the loop. If we want to keep the current approach of basing the feedback forwarding check on the reloaded identitiy notifications then we should move this outside the loop so that we only make one additional API call rather than 3.

The three topic settings won't affect each other so there should be no need to update between iterations of the loop.

But as I indicated in the other review comment, I think we're probably better checking the feedback forwarding state against the desired state of these topics rather than the reloaded identity_notifications since that way we avoid eventual consistency issues